web/src/components/storage/PartitionPage.tsx

```diff
@@ -512,56 +523,269 @@ type AutoSizeProps = {
   partition?: configModel.Partition;
 };
 
-function AutoSize({ mountPoint, partition }: AutoSizeProps): React.ReactNode {
-  const volume = useVolume(mountPoint);
-  const size = partition?.size;
-  const conditions: string[] = [];
-
-  if (volume.outline.snapshotsAffectSizes)
-    // TRANSLATORS: item which affects the final computed partition size
-    conditions.push(_("the configuration of snapshots"));
-
-  if (volume.outline.sizeRelevantVolumes && volume.outline.sizeRelevantVolumes.length > 0)
-    // TRANSLATORS: item which affects the final computed partition size
-    // %s is replaced by a list of mount points like "/home, /boot"
-    conditions.push(
-      sprintf(
-        _("the presence of the file system for %s"),
-        // TRANSLATORS: conjunction for merging two list items
-        volume.outline.sizeRelevantVolumes.join(_(", ")),
-      ),
+function autoSizeTextFallback(size): string {
+  if (size.max) {
+    if (size.max === size.min) {
+      return sprintf(
+        // TRANSLATORS: %s is a size with units, like "3 GiB"
+        _("A generic size of %s will be used for the new partition"),
```

Comment:
Why not to use `autoSizeTextFixed`? I would like to see the same sentence for "/home" and "/foo", for example: "A partition of 1 GiB will be created for /foo". I don't see the need of exposing the concept of "generic" size.

Reply:
For me is a very important concept to expose. When you are using a mount point that is described in the configuration of the product, then the installer is actually proposing a file system type and a range that somebody has configured for that mount point, so it makes sense an is (hopefully) a good and informed suggestion.

In the other case, those are just a generic (or "arbitrary" if you prefer) ranges and file system types. They may fit or not. We use the same for /var/log/whatever than for /opt or any other arbitrary thing. It's not really an informed decision so it should be taken less seriously.

Reply:
Moreover, I would even consider modifying the other sentence under discussion ("Let the installer propose a sensible range of sizes") for the fallback case and use something like "Use the default range of sizes"...

Reply:
So your problem is with the word "generic"?